pkg/protocol/http1/resp/header.go

```diff
@@ -96,6 +96,12 @@ func WriteHeader(h *protocol.ResponseHeader, w network.Writer) error {
 	return nil
 }
 
+// writeTrailer writes response trailer to w
+func WriteTrailer(h *protocol.ResponseHeader, w network.Writer) error {
```

Comment:
Can request and response use the same writetrailer func? It seems that this function only uses `Trailer` struct